lib/wise_homex/api_client_impl.ex

```diff
@@ -239,6 +239,10 @@ defmodule WiseHomex.ApiClientImpl do
     Request.post(config, "/devices/" <> device_id <> "/location", payload)
   end
 
+  def unset_device_location(config, device_id) do
+    Request.delete(config, "/devices/#{device_id}/location")
```

Comment:
Generally I have tried to use the `<>` operator for string concatenation in the api client. It is not completely consistent yet, however. It would be nice if that was used here too.

Reply:
Is this because of readability? I think `"/devices/#{device_id}/location"` is much more easy to read than `"/devices/" <> device_id <> "/location"`.

The first compiles to the same as `<<"/devices/", Kernel.to_string(device_id)::binary, "/location">>`, so there is an overhead, I think (The ekstra `to_string`) :)

Reply:
Feel free to use interpolation, if it is as good as concatenation. I agree that the readability is better using interpolation. Originally I did it believing there was a speed advantage, but if there isn't, we should just drop that convention.

Reply:
:+1: